app/models/manga.rb

```diff
@@ -86,16 +86,16 @@ def canonical_title
     romaji_title
   end
 
-  def poster_image_thumb
+  def poster_image_thumb(size = :large)
     if self.poster_image_file_name.nil?
       "https://hummingbird.me/assets/missing-anime-cover.jpg"
     else
       # This disgusting fastpath brought to you by the following issue:
       # https://github.com/thoughtbot/paperclip/issues/909
       if Rails.env.production?
-        "https://static.hummingbird.me/manga/poster_images/#{"%03d" % (self.id/1000000 % 1000)}/#{"%03d" % (self.id/1000 % 1000)}/#{"%03d" % (self.id % 1000)}/large/#{self.poster_image_file_name}?#{self.poster_image_updated_at.to_i}"
+        "https://static.hummingbird.me/manga/poster_images/#{"%03d" % (self.id/1000000 % 1000)}/#{"%03d" % (self.id/1000 % 1000)}/#{"%03d" % (self.id % 1000)}/#{size.to_s}/#{self.poster_image_file_name}?#{self.poster_image_updated_at.to_i}"
```

Comment:
File name should be escaped, and no idea why the poster image `large` style doesn't match anime. /cc @NuckChorris